active_learning/liver_tumor_al/active_learning.py

```diff
@@ -315,7 +315,7 @@ def main():
         unl_loader = DataLoader(unl_ds, batch_size=1)
 
         # Calculation of Epochs based on steps
-        max_epochs = np.int(args.steps / (np.ceil(len(train_d) / args.batch_size)))
+        max_epochs = int(args.steps / (np.ceil(len(train_d) / args.batch_size)))
```

Comment:
What are the benefits of changing to `np.int`?

Reply:
have reverted this change, thanks.